modules/govuk/manifests/app/nginx_vhost.pp

```diff
@@ -94,11 +94,19 @@
     $nginx_extra_config_real = $nginx_extra_config
   }
 
+  # Force HTTP basic auth for the training environment, since it doesn't have
```

Comment:
That would be ideal, but it seems like only 1 or 2 apps have a per-app configuration for HTTP basic auth. The main integration auth is done on the cache servers (https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_cache.pp#L52 and https://github.com/alphagov/govuk-puppet/blob/master/hieradata/integration.yaml#L197) which we don't have in the training environment. Do you know if there's a better way of doing this?